src/pybind/mgr/dashboard/frontend/src/app/shared/components/helper/helper.component.html

```diff
@@ -0,0 +1,8 @@
+<ng-template #popoverTpl><div [innerHtml]="html"></div></ng-template>
```

Comment:
Can't you use `ngContent `here? Instead of passing the html via `@Input`.
That way we could show more complex html.

Reply:
I've changed the implementation to support **both**.

1 - Useful when the helper text is stored on some controller property:

    this.helperHtml = 'This works';

    <cd-helper [html]="helperHtml">
    </cd-helper>

2 - Useful when the helper text is declared directly in the template:

    <cd-helper>
      <div cdHelperHtml>
        This also works
      </div>
    </cd-helper>